easybuild/easyconfigs/f/FastEpistasis/FastEpistasis-2.07-foss-2018a.eb

```diff
@@ -0,0 +1,38 @@
+#
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2021 University of Geneva Switzerland
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+#
+easyblock = 'CMakeMake'
+
+name = 'FastEpistasis'
+
+version = '2.07'
+
+homepage = 'https://gitlab.sib.swiss/tschuepb/FastEpistasis'
+
+description = '''
+   FastEpistasis, a software tool capable of computing tests of epistasis for a large number of SNP pairs, is an 
+   efficient parallel extension to the PLINK epistasis module.'''
+
+
```

Comment:
style nitpicking: please remove duplicate empty line

Reply:
Hi, this wasn't catch by `--check-contrib`. Is there any other check I should do before creating the PR or is it ok like that?

Reply:
We don't have a way of checking all code style issues currently.

`--check-contrib` is a very good start, but it's not complete. The easyconfigs test suite may complain about some things that `--check-contrib` doesn't.

In addition, some (very minor) code style issues like duplicate empty lines or order of easyconfig parameters won't be caught by either `--check-contrib` or the easyconfigs test suite.

From a technical point of view, these issues don't matter, but we try to maintain the same style and order of easyconfig parameters (more or less in order of execution), which helps humans when comparing easyconfig files.